fitbenchmarking/fitbenchmarking
# Combining benchmarking functions in core

#### Description of Work

Fixes #567 

#### Testing Instructions

1. Check the new structure is clearer

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
